src/protocol/templates/Ping1D.py.in

```diff
@@ -128,18 +124,16 @@ class Ping1D(object):
         return data
 
 {% endfor %}
-    # Control Methods
-    ###################
-    # Write a new control, and read the values back from the device
-    # If verify is true, fail if the values read back from the device do not match the controls that were written
-    # Return True on success
-    # Return False on failure to read values back or on verification failure
 
 {% for msg in messages["ping1D"]["set"] %}
+    ## Send a {{msg}} message to the device\n
```

Comment:
Why this `\n` is necessary ?

Reply:
Necessary to insert a newline (otherwise doxygen will ignore the invisible newline that's already there). Doxygen sucks with whitespace control to me. Let me know if there is a better way.

Reply:
I tried lots of ways to render the way I want without it. Can't get it without the \n